content/en/security/application_security/setup/go/setup.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Enabling App and API Protection for Go
+aliases:
+  - /security_platform/application_security/getting_started/go
+  - /security/application_security/getting_started/go
+  - /security/application_security/threats/setup/threat_detection/go
+  - /security/application_security/threats_detection/go
+further_reading:
+- link: "https://pkg.go.dev/github.com/DataDog/dd-trace-go/v2/appsec"
+  tag: "Documentation"
+  text: "Go Security API docs"
+- link: "/security/application_security/add-user-info/"
+  tag: "Documentation"
+  text: "Adding user information to traces"
+- link: 'https://github.com/DataDog/dd-trace-go'
+  tag: "Source Code"
+  text: 'Tracer source code'
+- link: 'https://github.com/DataDog/orchestrion'
+  tag: "Source Code"
+  text: 'Orchestrion source code'
+- link: "/security/default_rules/?category=cat-application-security"
+  tag: "Documentation"
+  text: "OOTB App and API Protection Rules"
+- link: "/security/application_security/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting App and API Protection"
+---
+
+You can monitor App and API Protection for Go apps running in Docker, Kubernetes, and Amazon ECS.
+
+{{% appsec-getstarted %}}
+
+# Prerequisite
+
+- Your service framework and tools are [compatible][2] with the [Application & API Protection (AAP)][1] product.
+- Your deployment environment is [supported][5].
+- You have one of the latest two version of [Go][4] installed (following the [Official Release Policy][5]).
+
+## Enabling Application & API Protection (AAP)
+
+### Get started
+
+1. **Install [Orchestrion][10]**:
+   ```console
+   $ go install github.com/DataDog/orchestrion@latest
+   ```
+
+2. **Register Orchestrion as a Go module** in your project directory:
+   ```console
+   $ orchestrion pin
+   ```
+
+3. Datadog has a series of pluggable packages which provide out-of-the-box support for instrumenting a series of Go libraries and frameworks.
+   A list of these packages can be found in the [compatibility requirements][1] page.
+   Import these packages into your application and follow the configuration instructions listed alongside each integration.
+
+4. **Recompile your program** with Orchestrion with the `appsec` build:
+   ```console
+   $ orchestrion go build -tags appsec my-program
```

Comment:
I like to have explicit `=` on flags here because it makes it clearer what goes with what.

```suggestion
   $ orchestrion go build -tags=appsec my-program
```

Reply:
It also may not be clear to a user what "my-program" is.